client/app/user/profile/profile.controller.js

```diff
@@ -85,6 +85,20 @@ angular.module('observatory3App')
           });
       };
 
+      $scope.pastUser = function(user){
+          var conf = confirm("Are you sure you want to make your account inactive?"); 
+          if (conf){
+            $http.put('/api/users/' + $stateParams.id + '/deactUser', {
```

Comment:
Should make use of the User factory instead of calling $http. You may need to add the deactivate functionality to the factory though